modules/mono/csharp_script.cpp

```diff
@@ -859,6 +870,15 @@ void CSharpLanguage::reload_assemblies(bool p_soft_reload) {
 		scr->_clear();
 	}
 
+	// Release the delegates that are where saved
```

Comment:
I don't understand what this is saying, is there a typo?

Reply:
*were saved in the // Serialize managed callables block